nettacker.py

```diff
@@ -13,6 +13,26 @@
 
     # if dependencies and OS requirements are match then load the program
     from core.parse import load
+    import sys
 
-    load()  # load and parse the ARGV
+    # Example commands to demonstrate OWASP Nettacker functionalities
+    example_commands = [
+        "-i target.com",
```

Comment:
Just FYI `target.com` is a real company web-site that may not appreciate to be a default example even in a pretty small audience penetration testing tool.

Reply:
@arkid15r I will take care of this and add more info to --help and the new documentation. The examples in this PR are not 1005 correct as they are using non-existing modules and switches, but I get the idea

Reply:
> @arkid15r I will take care of this and add more info to --help and the new documentation. The examples in this PR are not 1005 correct as they are using non-existing modules and switches, but I get the idea

Sounds good, thanks Sam!